src/chigame/games/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import apps
+from django.contrib import admin
+
+app_models = apps.get_app_config("games").get_models()
+for model in app_models:
+    admin.site.register(model)
```

Comment:
While this syntax for registering models works, I want you to register the models individually. Since this way we can much more easily apply admin interface options for each model.

**For this PR, remove this code**, since @ptlin25 has his task about registering the game model where this should be addressed.

Checkout the below links to see more about the customization of the admin interface:

https://docs.djangoproject.com/en/4.2/ref/contrib/admin/

https://realpython.com/customize-django-admin-python/

Reply:
@fkbad Got it, thanks for the feedback! I've made this change in the newest commit, please let me know what you think.